pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -183,28 +183,11 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 	logger.Debug("PA exists")
 
 	desiredMetric := resources.MakeMetric(ctx, pa, c.dynConfig.Current())
```

Comment:
I think this line should move into `kpaMetric/reconcileMetric`.

Reply:
Done.